src/main/java/core/basesyntax/model/OperationIstrategy.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.model;
+
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.storage.StorageDao;
+import core.basesyntax.strategy.impl.BalanceOperationHandler;
+import core.basesyntax.strategy.impl.PurchaseOperationHandler;
+import core.basesyntax.strategy.impl.ReturnOperationHandler;
+import core.basesyntax.strategy.impl.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperationIstrategy {
```

Comment:
```suggestion
public class OperationStrategy {
```
сорі, в коментарі була опічатка